src/AutoMapper/ExpressionExtensions.cs

```diff
@@ -69,5 +69,14 @@ public static Expression NullCheck(this Expression expression, Type destinationT
 
         public static Expression IfNullElse(this Expression expression, Expression then, Expression @else = null)
             => ExpressionFactory.IfNullElse(expression, then, @else);
+
+        public static LambdaExpression[] WithoutCastToObject<T>(this Expression<Func<T, object>>[] expressions)
```

Comment:
This is used in just one place, so just make it a local function at the call site.

Reply:
You didn't move this.

Reply:
What do you mean? I removed it from the ExpressionExtensions.cs file, which is now untouched by this PR. The code is now in the only method that uses it.

Reply:
_Now_  it's gone :)